pinot-spi/src/main/java/org/apache/pinot/spi/data/DateTimeFormatPatternSpec.java

```diff
@@ -65,6 +66,21 @@ public DateTimeFormatPatternSpec(String timeFormat, @Nullable String sdfPatternW
     }
   }
 
+  public DateTimeFormatPatternSpec(DateTimeFieldSpec.TimeFormat timeFormat, String timeFormatStyle,
+      @Nullable String timeZone) {
+    _timeFormat = timeFormat;
+    if (_timeFormat.equals(DateTimeFieldSpec.TimeFormat.SIMPLE_DATE_FORMAT)) {
+      if (Objects.isNull(timeZone)) {
+        _dateTimeZone = DateTimeZone.UTC;
```

Comment:
You can use the `DEFAULT_DATETIMEZONE` variable here already declared in class

Reply:
Changed in the latest commit